packages/react-native-reanimated/src/logger/logger.ts

```diff
@@ -57,4 +72,4 @@ function createLogger(prefix: string) {
   };
 }
 
-export const logger = createLogger('[Reanimated]');
+global.__reanimatedLogger = createLogger('[Reanimated]');
```

Comment:
I wonder if we actually want to add this to the global scope. It's definitely doable to just export it with some dependency inversion.

Also, are we certain that the logger will be defined in the global scope already for every case where we use it?